src/main.js

```diff
@@ -115,8 +115,10 @@ function saveButtonDisable() {
 function searchFilter() {
   allCards.innerHTML = "";
   for (var i = 0; i < storage.length; i++) {
-    if (storage[i].title.includes(searchInput.value) ||
-      storage[i].body.includes(searchInput.value)) {
+    var title = storage[i].title.toLowerCase()
```

Comment:
Would this work on toUpper? 
I am glad we made this catch on this bug 
```suggestion
    var title = storage[i].title.toLowerCase()
```